addons/test_mail/tests/test_performance.py

```diff
@@ -267,17 +267,17 @@ def test_mail_composer(self):
                 'partner_ids': [(4, customer_id)],
             })
 
-        with self.assertQueryCount(__system__=36, emp=41):
+        with self.assertQueryCount(__system__=31, emp=37):
```

Comment:
a multibuild would be great to validate query count changes

Reply:
Hopefully the only remaining issue is the query that is marked as non deterministic, for which I have been extra careful.

If there are others discovered since this merge feel free to ping me and I'll handle them.